frontend/pages/donate-now/Error.jsx

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import React from "react";
+import Layout from "../../components/Layout";
+
+class Error extends React.Component {
```

Comment:
@sunzae99, for consistency with the rest of our frontend codebase, can you use function-based components?

It won't require changing much.  You can look at frontend/pages/entities/index.jsx for an example in JavaScript or (even better) frontend/pages/auth/redirect.tsx for an example in TypeScript.